corehq/apps/export/templates/export/partials/table.html

```diff
@@ -74,6 +74,231 @@
+                        </div>
+                    {% endangularjs %}
+                    {% angularjs %}
+                    <div class="modal fade"
+                             id="modalEnableDisableAutoRefresh-{{ export.id }}-{{ export.emailedExport.groupId }}">
+                            {% endangularjs %}
+                            <div class="modal-dialog">
+                                <div class="modal-content">
+                                    <div class="modal-header">
+                                        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">{% trans 'Close' %}</span></button>
+                                        <h4 class="modal-title">{% trans "Confirm data updates." %}</h4>
+                                    </div>
+                                    <div class="modal-body">
+                                        <p ng-if="export.isDailySaved && !export.isAutoRebuildEnabled">
+                                            {% blocktrans %}
+                                            This will enable the daily automatic updates of this export.
+                                            {% endblocktrans %}
+                                        </p>
+                                        <p ng-if="export.isDailySaved && export.isAutoRebuildEnabled">
+                                            {% blocktrans %}
+                                            This will disable the daily automatic updates of this export.
+                                            {% endblocktrans %}
+                                        </p>
+                                    </div>
+                                    <div class="modal-footer">
+                                        <a href="#"
+                                           class="btn btn-default"
+                                           data-dismiss="modal">{% trans "Cancel" %}</a>
+                                        <button type="button"
+                                                class="btn btn-primary"
+                                                ng-click="updateDisabledState(export.emailedExport, export)">
+                                            <span ng-if="export.isAutoRebuildEnabled">
+                                                {% trans "Disable auto update" %}
+                                            </span>
+                                            <span ng-if="!export.isAutoRebuildEnabled">
+                                                {% trans "Enable auto update" %}
+                                            </span>
+                                        </button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </td>
+ 
+
```

Comment:
If someone ever asks me why copy/pasting code is bad, I'm gonna show them this commit